packages/ui/src/overview/components/DeadlineList/ElectionListItem.tsx

```diff
@@ -83,7 +84,13 @@ export const ElectionListItem: React.FC<ElectionListItemProps> = React.memo(({ e
     <ElementWrapper>
       <ListItem>
         <TopElementsWrapper>
-          <StyledTriangle deadlineTime={remainingCalculation} /> <StyledClosedButton onClick={setHideElement} />
+          <StyledTriangle deadlineTime={remainingCalculation} />{' '}
+          <StyledClosedButton
+            onClick={() => {
+              setHideElement()
```

Comment:
`useLocalStorage` uses `useState` to update the data in real time (it doesn't just store data and retrieve them when mounted). As a result this shouldn't be needed anymore